server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -269,6 +270,13 @@ public boolean isLeader()
   )
   {
     final Map<String, Object2LongMap<String>> underReplicationCountsPerDataSourcePerTier = new HashMap<>();
+    final Set<String> decommissioningServers = getDynamicConfigs().getDecommissioningNodes();
```

Comment:
In a previous PR there was a discussion about why it's ok for `segmentReplicantLookup` to be stale in this method:  https://github.com/apache/druid/pull/9965/files#r440541949

What do you think about having that explanation as a code comment for this method?

Reply:
Done